### PR TITLE
Article on median of two arrays

### PR DESCRIPTION
**Pull request by Amit366**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Added Article on median of two arrays

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes: #299

### CODE REVIEW

**Comment by Amit366** [comment]
@iamrajiv please review

**Comment by Amit366** [comment]
@iamrajiv please can help me to resolve the test error??

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Amit366 I don't think `fractional-knapsack.mdx` belongs to this PR.

**Comment by Amit366** [comment]
@iamrajiv I have removed the fractional knapsack file

File: docs/dynamic-programming/median-of-two-arrays.mdx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    }
+
+    /* Sort the given vector */
+    sort(v.begin(), v.end());
+    
+    /* Find medina using this formula */
+    int mid = v[v.size() / 2];
+    
+    /* Return Calculated median */
+    return mid;
```
Comment (ankitaggarwal23) [inline docs/dynamic-programming/median-of-two-arrays.mdx:111]: Use single-line comments instead multiline

File: docs/dynamic-programming/median-of-two-arrays.mdx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+A | -5 | 3 | 6 | 12 | 15 |
+B | -12 | -10 | -6 | -3 | 4 | 10 |
+
+* Now in a third array merge the `A` and `B`.
+
+C | -5 | 3 | 6 | 12 | 15 | -12 | -10 | -6 | -3 | 4 | 10 |
+
+* Now sort C
+
+C | -12 | -10 | -6 | -5 | -3 | 3 | 4 | 6 | 10 | 12 | 15 |
```
Comment (ankitaggarwal23) [inline docs/dynamic-programming/median-of-two-arrays.mdx:33]: It would be great if you could should this using a diagrammatic representation

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by Amit366** [comment]
@ankitaggarwal23 please review

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Don't add phone clicked image you can use the snip tool for taking a screenshot on desktop and Don't add screenshot all add once as you are doing step wise

**Comment by Amit366** [comment]
@ankitaggarwal23 now is it fine??

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**